### PR TITLE
drip: init at 0.2.4

### PR DESCRIPTION
**Pull request by rybern**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by rybern** [comment]
Not totally sure if this belongs in development/tools, but I didn't know where would be better. It's a wrapper around the 'java' command.

**Comment by rybern** [comment]
What does the failing check mean, and how can I fix it?

File: pkgs/development/tools/drip/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  buildPhase = ''
+    $out/bin/drip version
+  '';
+
+  phases = [ "unpackPhase" "patchPhase" "buildPhase" ];
+
+  meta = with stdenv.lib; {
+    description = "a launcher for the Java Virtual Machine intended to be a drop-in replacement for the java command, only faster";
+    license = licenses.epl;
```
Comment (pbogdan) [inline pkgs/development/tools/drip/default.nix]: This should be `licenses.epl10`, `epl` attribute doesn't exist and I suspect is what made the check fail.

File: pkgs/development/tools/drip/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, gcc, jdk, which }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "drip";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "flatland";
```
Comment (pbogdan) [inline pkgs/development/tools/drip/default.nix]: Not that I know anything about this particular package but is there a reason to use a fork that appears to be behind the parent repo?

**Comment by grahamc** [comment]
@grahamcofborg eval

**Comment by grahamc** [comment]
@rybern, the test failures were likely caused by a commit on master, and not this branch. I've triggered them to try again.

File: pkgs/development/tools/drip/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  postPatch = ''
+    patchShebangs .
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir $out
+    cp ./* $out -r
+    wrapProgram $out/bin/drip \
+      --prefix PATH : "${which}/bin"
```
Comment (Mic92) [inline pkgs/development/tools/drip/default.nix]: I wonder if `jre` should also be there or if this should be up to the user to decide.

**Comment by Mic92** [comment]
@GrahamcOfBorg build drip

**Pull request merged**